### PR TITLE
add a few deinflections for Latin

### PR DESCRIPTION
**Pull request by StefanVukovic99**
I don't think I'll be finishing this in the foreseeable future, but it provides some utility and someone can add to it easily.

### CODE REVIEW

**Comment by jamesmaa** [comment]
I don't know Latin do you still want me to approve this?

**Comment by StefanVukovic99** [comment]
Maybe @martholomew can give it a lookover, but it can't break anything so it's safe to merge :pray:

File: test/language/latin-transforms.test.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import {latinTransforms} from '../../ext/js/language/la/latin-transforms.js';
+import {LanguageTransformer} from '../../ext/js/language/language-transformer.js';
+import {testLanguageTransformer} from '../fixtures/language-transformer-test.js';
+
+/* eslint-disable @stylistic/no-multi-spaces */
```
Comment (jamesmaa) [inline test/language/latin-transforms.test.js:23]: What is this for?

**Review by jamesmaa — APPROVED** [review]

File: test/language/latin-transforms.test.js (hunk 1 above)
Comment (StefanVukovic99) [reply test/language/latin-transforms.test.js:23]: Usually spaces are used to align everything nicely there, might not have done that there but kept the comment

Comment (StefanVukovic99) [reply test/language/latin-transforms.test.js:23]: I cleaned up the spaces and the missing reasons.

**Review by jamesmaa — APPROVED** [review]

**Pull request merged**